Update release-notes.md (#128)
tidepool-org/tidepool-org.github.io: _docs/release-notes.md

Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  

### 1.23.0 (Released 2019-07-05)

> We're starting off July with a bugfix bonanza. First, we cleaned up some spacing and formating when you edit your profile.  

Reviewer: ```suggestion
We're starting off July with a bugfix bonanza. First, we cleaned up some spacing and formatting when you edit your profile.  
```

If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  

### 2.17.0 (Released 2019-06-25)

> Tidepool Uploader now supports the Omnipod DASH PDM. Woohoo! Whether you're uploading the original PDM or the DASH PDM, you'll still select "Insulet Omnipod" as your device we'll take care of the rest.  

Reviewer: ```suggestion
Tidepool Uploader now supports the Omnipod DASH PDM. Woohoo! Whether you're uploading the original PDM or the DASH PDM, you'll still select "Insulet Omnipod" as your device and we'll take care of the rest.  
```

### 2.14.0 (Released 2019-04-26)
Often requested by our users, we now display scan events from the FreeStyle Libre in Tidepool Web. Now you can see when you're scanning that sensor, and what the value was for each scan.  
[Feature Requirements](https://trello.com/c/O9i3tqCA/60-uploader-upload-libre-scan-records-as-smbg-record)

We updated the error message that displays when a clinician tries to create a Tidepool account with an email address already associated with an existing Tidepool account. Little things like that are important.  
[Feature Requirements](https://trello.com/c/7HVJ9gUH/52-bug-uploader-not-handling-the-user-already-exists-error)

Continuing the user experience improvement train, the design of our Mac Installer has improved to be more explicit about the "drag this app into that folder" bit.  
[Feature Requirements](https://trello.com/c/LpB6l0Sd/53-improve-usability-of-mac-dmg-installer)

And now we're rolling out more test environments to improve our development process, development builds of Tidepool Uploader need to access them.  
[Feature Requirements](https://trello.com/c/2wTVDU61/54-uploader-add-qa1-environment-to-uploader-server-list)

> We created a script that automatically submits a notification to Anti-virus vendors like McAffe and Kaspersky when new versions of Tidepool Uploader are released.  

Reviewer: ```suggestion
We created a script that automatically submits a notification to antivirus vendors like McAfee and Kaspersky when new versions of Tidepool Uploader are released.  
```

[Feature Requirements](https://trello.com/c/Wv9TE21v/55-update-script-to-whitelist-with-kaspersky-and-mcafee-automatically)

> We now store Omnipod IBF files in our "blog" service, which helps tremendously when we are troubleshooting reported bugs from our users. (We already do this for Medtronic uploads.)  

Reviewer: ```suggestion
We now store Omnipod IBF files in our "blob" service, which helps tremendously when we are troubleshooting reported bugs from our users. (We already do this for Medtronic uploads.)  
```

## Tidepool Mobile  
Visit https://tidepool.org/mobile to learn more about Tidepool Mobile.  

### iOS - 2.1.12 (Released 2019-07-02)
We added a tooltip to prompt new Tidepool Mobile users to make a note after they've synced their data.  

> [Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-9?jql=project%20%3D%20%22MOBILE%22%20AND%20fixVersion%20%3D%20%22iOS%202.1.12%22)

Reviewer: ```suggestion
[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-9)
```

And addressed a design issue that prevented users from opening a note created as a result of that first time tooltip prompt.  

> [Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-10?jql=project%20%3D%20%22MOBILE%22%20AND%20fixVersion%20%3D%20%22iOS%202.1.12%22)

Reviewer: ```suggestion
[Feature Requirements](https://tidepool.atlassian.net/browse/MOBILE-10)
```

[iOS Release](https://appsto.re/us/aXyl9.i)

### Android - 3.0.5 (Released 2019-06-13)

> Technically this work also supports our eventual iOS update to a React Native code base soe both Android and iOS versions of Tidepool Mobile will be using the same code base. This update continues to iterate upon the tooltips that display for fresh installs. We'll get to updating the current iOS release with these tooltips soon.  

Reviewer: ```suggestion
Technically this work also supports our eventual iOS update to a React Native code base so both Android and iOS versions of Tidepool Mobile will be using the same code base. This update continues to iterate upon the tooltips that display for fresh installs. We'll get to updating the current iOS release with these tooltips soon.  
```
